tests/integration/test_demo_functionality.py

```
@@ -0,0 +1,242 @@
+        expected_request = {
+            "audio_data": sample_audio_data,
+            "model": "whisper"
+        }
+
+        # Validate the request format
+        assert "audio_data" in expected_request
+        assert "model" in expected_request
+        assert expected_request["model"] == "whisper"
+
+    @staticmethod
+    def test_demo_whisper_invalid_audio(demo_api_url):
```

Ruff (0.12.2)

Unused static method argument: `demo_api_url`

(ARG004)